docs/document/content/others/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md

```diff
@@ -3,7 +3,7 @@ title = "Change History"
 weight = 7
```

Comment:
weight should be equal to '4', 
please refer to https://github.com/apache/shardingsphere/tree/master/docs/document/content/others/api-change-history/shardingsphere-jdbc